x-pack/filebeat/input/cel/input.go

```diff
@@ -100,7 +106,12 @@ func (i input) now() time.Time {
 func (input) Name() string { return inputName }
 
 func (input) Test(src inputcursor.Source, _ v2.TestContext) error {
-	cfg := src.(*source).cfg
+	srcp, ok := src.(*source)
+	if !ok {
+		return fmt.Errorf("input type %T is not a source", src)
+	}
```

Comment:
Please don't do this. If this panics something has gone terribly wrong and the program should fail fatally. If absolutely necessary, you can add a `//nolint:errcheck // If this assertion fails, the program is incorrect and should panic.`.